cypress/stubs/orgSignUp/mappings/api_v1_users-d2c41957-0331-4c20-a597-7002630980ab.json

```diff
@@ -5,14 +5,14 @@
     "url" : "/api/v1/users?activate=false",
     "method" : "POST",
     "bodyPatterns" : [ {
-      "equalToJson" : "{\"profile\":{\"firstName\":\"Greg\",\"lastName\":\"McTester\",\"login\":\"greg@mailinator.com\",\"email\":\"greg@mailinator.com\"},\"groupIds\":[\"00g228nyd7uuGaG1L1d7\",\"00g228nrj9QDJ10y61d7\"]}",
+      "equalToJson" : "{\"profile\":{\"firstName\":\"Greg\",\"lastName\":\"McTester\",\"login\":\"${json-unit.any-string}\",\"email\":\"${json-unit.any-string}\"},\"groupIds\":[\"00g228nyd7uuGaG1L1d7\",\"00g228nrj9QDJ10y61d7\"]}",
```

Comment:
this is a neat trick!

Reply:
thought I'm realizing this is in an auto-generated file 🤣

Reply:
@georgehager mostly auto-generated! It's created by [WireMock](https://github.com/CDCgov/prime-simplereport/wiki/WireMock) but this `any-string` piece is hand-touched :)

Reply:
This was a manual update to allow for any email. Doing so lets us add more people and not need to clean the db every time.

Reply:
Neat! I saw in the test we type in `user.email` so that makes sense!